src/config-utils.ts

```diff
@@ -933,6 +948,19 @@ async function loadConfig(
     }
   }
 
+  let dbLocation;
+  if (DB_LOCATION in parsedYAML) {
+    if (typeof parsedYAML[DB_LOCATION] !== "string") {
+      throw new Error(dbLocationInvalid(configFile));
+    }
+    if (parsedYAML[DB_LOCATION]!.length === 0) {
+      throw new Error(dbLocationInvalid(configFile));
+    }
```

Comment:
Nit: to avoid the type assertion `!.length`.

```suggestion
    if (typeof parsedYAML[DB_LOCATION] !== "string" || parsedYAML[DB_LOCATION] == "") {
      throw new Error(dbLocationInvalid(configFile));
    }
```

Reply:
(No longer applicable given we are changing it to an Actions input per below)